doc/nrf/ug_multi_image.rst

```diff
@@ -129,58 +132,113 @@ For example:
+   Child images that are built with the build strategy ``<IMAGE_NAME>\ _BUILD_STRATEGY_SKIP_BUILD`` or ``<IMAGE_NAME>\ _BUILD_STRATEGY_USE_HEX_FILE`` must define a :ref:`static partition <ug_pm_static_providing>`.
 
+Add these configuration options to the Kconfig file of your child image, replacing ``<IMAGE_NAME>`` with the uppercase name of your child image, as specified in ``add_child_image``.
 
-You must add these four configuration options to the Kconfig file for your child image, replacing *IMAGENAME* with the (uppercase) name of your child image (as specified in ``add_child_image``).
-
-This can be done by including the :file:`Kconfig.template.build_strategy` template, as shown below.
+This can be done by including the :file:`Kconfig.template.build_strategy` template as follows:
 
 .. code-block:: Kconfig
 
    module=MCUBOOT
    source "${ZEPHYR_NRF_MODULE_DIR}/subsys/partition_manager/Kconfig.template.build_strategy"
 
+.. _ug_multi_image_variables:
 
 Image-specific variables
 ========================
 
-The child image and parent image are executed in different CMake processes and thus have different namespaces.
+The child and parent images are executed in different CMake processes and thus have different namespaces.
+
 Variables in the parent image are not propagated to the child image, with the following exceptions:
 
-* Any variable named *IMAGENAME*\_FOO in a parent image is propagated to the child image named *IMAGENAME* as FOO.
-* Variables that are in the list ``SHARED_MULTI_IMAGE_VARIABLES`` are propagated to all child images.
+* Any variable named ``<IMAGE_NAME>\_FOO`` in a parent image is propagated to the child image named ``<IMAGE_NAME>`` as ``FOO``.
+* CMake build settings, such as ``BOARD_DIR``, build type, toolchain info, partition manager info, and similar are always passed to child images.
 
 With these two mechanisms, it is possible to set variables in child images from either parent images or the command line, and it is possible to set variables globally across all images.
-For example, to change the ``CONF_FILE`` variable for the MCUboot image and the parent image, specify the CMake command as follows::
+For example, to change the ``CONF_FILE`` variable for the MCUboot image and the parent image, specify the CMake command as follows:
+
+.. code-block::
 
    cmake -Dmcuboot_CONF_FILE=prj_a.conf -DCONF_FILE=app_prj.conf
 
 You can extend the CMake command that is used to create the child images by adding flags to the CMake variable ``EXTRA_MULTI_IMAGE_CMAKE_ARGS``.
 For example, add ``--trace-expand`` to that variable to output more debug information.
 
+With ``west``, these configuration variables are passed into CMake by using the ``--`` separator:
+
+.. code-block::
+
+   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- -Dmcuboot_CONF_FILE=prj_a.conf -DCONF_FILE=app_prj.conf
+
+Variables in child images
```

Comment:
@tejlmand @kyberdin I just added the section about variables in child images, as mentioned in one of the previous comments.